api/main.go

```diff
@@ -44,11 +44,14 @@ func main() {
 	// Account specific actions
 	a := r.PathPrefix("/me").Subrouter().Inject(handlers.JwtTokenValidatorHandler)
 	a.HandleFunc("/validate", handlers.Validate).Methods("POST")
-	a.HandleFunc("/save", handlers.Save).Methods("POST")
+	a.HandleFunc("/save", handlers.Save).Methods("POST", "PUT")
```

Comment:
does this mean both HTTP methods do the same thing?

Reply:
Correct. We have one endpoint which handles creation and updating of the data.